flutter-studio/src/io/flutter/profiler/ResetStatsDataSeries.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.profiler;
+
+import com.android.tools.adtui.model.DataSeries;
+import com.android.tools.adtui.model.Range;
+import com.android.tools.adtui.model.SeriesData;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.jetbrains.annotations.NotNull;
+
+public final class ResetStatsDataSeries implements DataSeries<ResetData> {
+  FlutterAllMemoryData.ThreadSafeData resetData;
+
+  public ResetStatsDataSeries(FlutterAllMemoryData.ThreadSafeData gcDataSeries) {
+    resetData = gcDataSeries;
+  }
+
+  @Override
+  public List<SeriesData<ResetData>> getDataForXRange(@NotNull Range xRange) {
+    List<SeriesData<ResetData>> seriesData = new ArrayList<>();
+
+    List<SeriesData<Long>> rawGcValues = resetData.getDataForXRange(xRange);
+    for (SeriesData<Long>  sample : rawGcValues) {
```

Comment:
make sure the code formatter has been run on all code. This indentation seems a bit non-standard.

Reply:
re-ran formatter.